Selenium.Axe/Selenium.Axe.Test/AxeBuilderTest.cs

```diff
@@ -11,6 +12,36 @@ namespace Selenium.Axe.Test
     [TestClass]
     public class AxeBuilderTest
     {
+        private Mock<IWebDriver> webDriverMock;
+        private Mock<IJavaScriptExecutor> jsExecutorMock;
+        private Mock<ITargetLocator> targetLocatorMock;
+        private static readonly JsonSerializerSettings JsonSerializerSettings = new JsonSerializerSettings
+        {
+            Formatting = Formatting.None,
+            NullValueHandling = NullValueHandling.Ignore
+        };
+
+        private readonly string testAxeResult = JsonConvert.SerializeObject(new
```

Comment:
Part of the test's job is to verify that it correctly deserializes axe-like output; I think it would be a better test to use a string here, rather than calling `JsonConvert.SerializeObject`